src/darwin/Framework/CHIP/MTRIMDispatch.mm

```diff
@@ -121,6 +121,11 @@ CHIP_ERROR ReadSingleClusterData(const SubjectDescriptor & aSubjectDescriptor, b
         return aAttributeReports.EncodeAttributeStatus(aPath, StatusIB(status));
     }
 
+    bool ConcreteAttributePathExists(const AttributePathParams & aPath)
+    {
+        return DetermineAttributeStatus(aPath, /* aIsWrite = */ false) == Status::Success;
```

Comment:
Just realized this bit is here.  This is not really right, not least because `DetermineAttributeStatus` _always_ returns a failure status.  This should probably be more like:
```suggestion
        return DetermineAttributeStatus(aPath, /* aIsWrite = */ false) == Status::UnsupportedAccess;
```